modules/tinymce/src/themes/silver/main/ts/backstage/Backstage.ts

```diff
@@ -51,7 +53,8 @@ const init = (lazySink: () => Result<AlloyComponent, string>, editor: Editor, la
         menuItems: () => editor.ui.registry.getAll().menuItems,
         translate: I18n.translate,
         isDisabled: () => editor.mode.isReadOnly() || !editor.ui.isEnabled(),
-        getOption: editor.options.get
+        getOption: editor.options.get,
+        tooltips: TooltipsBackstage(lazySink)
```

Comment:
Because we are ending up with a lot of duplication with translations etc., it might even be worth passing translate in here and doing it there. We'd then just have to make sure we aren't double-translating.